var/spack/repos/builtin/packages/byfl/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Byfl(CMakePackage):
+    """Compiler-based Application Analysis"""
+
+    homepage = "https://github.com/lanl/Byfl"
+    url = "https://github.com/lanl/Byfl"
```

Comment:
I believe this should be
```Python
    url = "https://github.com/lanl/Byfl/archive/refs/tags/v1.8.0.tar.gz"
```
Otherwise, Spack complains that
```
==> Error: Unable to parse extension from https://github.com/lanl/Byfl.

If this URL is for a tarball but does not include the file extension
in the name, you can explicitly declare it with the following syntax:

    version('1.2.3', 'hash', extension='tar.gz')
```